quickstarts/elixir.mdx

```diff
@@ -0,0 +1,309 @@
+
+The workflow includes the following:
+
+- Workflow name and an Identifier.
+- Channel-tailored content:
+
+| Channel | Content Style |
+| --- | --- |
+| Email | 1. Custom Code (HTML) with option to use custom variables via the handlebars syntax. |
+|  | 2. Click and place UI items with the visual template editor. |
+| SMS | Text with the option to use handlebars syntax to inject custom variables. |
+| Chat | Text with the option to use handlebars syntax to inject custom variables. |
+| In-App | Text |
+
+<Note>
+Proper authorization needs to be set for the Chat channel for subscribers.
+</Note>
+
+Now, proceed to create a workflow as follows:
+
+1. Click **Workflows** on the left sidebar of your Novu dashboard.
+2. Click on the **Create Workflow** button on the top right.
+<Frame caption="Use the create workflow button to create a workflow"><img src="/images/Quickstart_Elixir/1.png" /> </Frame>
+3. The name of the new workflow is currently "Untitled", rename it to a more suitable title.
+<Frame caption="Rename the workflow to a more suitable title"><img src="/images/Quickstart_Elixir/2.png" /> </Frame>
+4. Select Email as the channel you want to add, by dragging it from the right sidebar.
+<Frame caption="Select Email as the channel you want to add, by dragging it from the right sidebar."><img src="/images/Quickstart_Elixir/3.png" /> </Frame>
+5. Click on the Email in the workflow and edit it as per this image. Don’t forget to add the fields in the editor which is supposed to be updated with dynamic values that will be sent when calling the API.
+<Frame caption="Add the fields in the email editor"><img src="/images/Quickstart_Elixir/4.png" /> </Frame>
+6. Also, add the variables in the Variables section in the test tab and try testing it by sending the email to your email address using the Send Test Email button at the bottom right.
+<Frame caption="Add the variables in the Variables section in the test tab"><img src="/images/Quickstart_Elixir/5.png" /> </Frame>
+
+You should be able to get an email within seconds! 👌
+Now, you have successfully sent your first notification with the Novu dashboard, let’s move to a next step to trigger various notifications via code.
+
+## Create a Subscriber and Update a Subscriber's information
+
+The recipients of a triggered notification are called subscribers.
+
+Click **Subscribers** on the left sidebar of the Novu dashboard to see all subscribers. By default, the dashboard will display a subscriber, as you were added automatically during sign-up.
+<Frame caption="By default, the dashboard will display a subscriber, as you were added automatically during sign-up."><img src="/images/default-sub-kotlin.png" /> </Frame>
+Now, let's create a subscriber on Novu. Copy and paste the following code to do so:
+
+```elixir
+
```

Comment:
I can't see the Novu module or class been imported anywhere in the code samples. @btkostner is this right? Please help review.

Reply:
This code is incorrect. The `Novu.create_subscriber/2` function should follow the [novu-elixir library function call here](https://github.com/novuhq/novu-elixir/blob/main/lib/novu/subscribers.ex#L28) where the payload can be configured by passing in opts as describe by [this function](https://github.com/novuhq/novu-elixir/blob/main/lib/novu/subscribers.ex#L172). Secondly, the `Task.await` and `Task.async` calls here are pointless and should be removed.

Instead of declaring a module, I'd write this more of a script and just do something like:

```elixir
{:ok, response} = Novu.create_subscriber("12345", [
  email: "email@sample.com",
  first_name: "John",
  last_name: "Doe",
  phone: "123456789",
  avatar: "sample-avatar",
])
```